source/mainnet/net/guides/create-identity.rst

```diff
@@ -82,6 +82,10 @@ Before you can start using a Wallet and submit transactions to the blockchain, y
 
    .. tab:: |mw-gen1|
 
+        .. Note::
```

Comment:
The same goes here. I'm not sure if it makes sense to explain how to create an identity below, if it can't be done anyway.

Reply:
Removed